### PR TITLE
Use new notification style for Android O and above

### PR DESCRIPTION
**Pull request by KristupasSavickas**

### CODE REVIEW

File: app/release/output.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+  "version": 1,
+  "artifactType": {
+    "type": "APK",
+    "kind": "Directory"
+  },
+  "applicationId": "com.dkanada.gramophone",
+  "variantName": "release",
```
Comment (dkanada) [inline app/release/output.json:8]: Is this file required?

Comment (KristupasSavickas) [reply app/release/output.json]: Sorry, removed.

**Comment by KristupasSavickas** [comment]
Marked as a draft as I'm not sure how this will affect older versions. Will convert to a normal PR once I have the chance to test.

**Comment by KristupasSavickas** [comment]
Android N was weird, the "new" notification wasn't getting colorized like it does for the more recent Android versions. So I changed the threshold to O. Here are the previews across different versions with this PR:
## Lollipop
Lollipop
## Marshmallow
Marshmallow
## Nougat
Nougat
## Oreo
Oreo
## Pie
Pie
## Q
Q

**Comment by dkanada** [comment]
@KristupasSavickas just to be clear, are the photos before or after the change in your pull request?

**Comment by KristupasSavickas** [comment]
> @KristupasSavickas just to be clear, are the photos before or after the change in your pull request?

After

**Comment by dkanada** [comment]
Thanks for testing all major versions! It takes all the guesswork out of version support.
